### DISCUSSION TITLE
Don't add values to loop unless they're within a certain range?

### DISCUSSION THREAD

**Discussion by kevinkey619** [bellrichm/WeeWX-MQTTSubscribe#158] [General]
Is there a way to tell the logic to NOT add blank data values to the loop? I'm having a problem where blank pressure values are getting added when the loop interval doesn't coincide with the MQTT update interval. Do I need to uninstall and reinstall? Been spending the last few weeks trying to figure out what I'm doing wrong? Do I need to reconfigure the MQTT publishing service to publish the pressure more than once every second?

**Comment by bellrichm** [comment]
Lets work through your problems in one place, https://github.com/bellrichm/WeeWX-MQTTSubscribe/discussions/157
